### PR TITLE
EDU-1552 Add document categories tab

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1552

### CODE REVIEW

File: src/domain/validation-constants.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ export const maxDocumentCommentTextLength = 2000;
 
 export const maxDocumentRevisionCreatedBecauseLength = 200;
 
+export const maxDocumentCategoryNameLength = 25;
```
Comment (ahelmberger) [inline src/domain/validation-constants.js:10]: This we should also use in the `documentCategoryDbSchema` validation (which is not included in your branch yet). Just as a reminder.

File: src/components/icons/icons.js [hunk 2]
```diff
@@ -81,6 +86,7 @@ export const FillColorIcon = createIconFromTablerIconComponent(IconBucketDroplet
 export const ChevronLeftIcon = createIconFromTablerIconComponent(IconChevronLeft);
 export const EnterFullscreenIcon = createIconFromTablerIconComponent(IconMaximize);
 export const ChevronRightIcon = createIconFromTablerIconComponent(IconChevronRight);
+export const MediaLibraryIcon = createIconFromTablerIconComponent(IconBuildingBank);
```
Comment (ahelmberger) [inline src/components/icons/icons.js:89]: :+1:

**Review by ahelmberger — APPROVED** [review]

File: src/domain/validation-constants.js (hunk 1 above)
Comment (adrianaluchian) [reply src/domain/validation-constants.js:10]: Good observation, I rebased from `main` and included it there too 👍

Comment (ahelmberger) [reply src/domain/validation-constants.js:10]: That is even better than just a reminder :+1:

**Pull request merged**